src/javascript/app/pages/user/account/settings/personal_details.js

```diff
@@ -98,11 +175,12 @@ const PersonalDetails = (() => {
     };
 
     const show_label_if_any_value   = ['account_opening_reason', 'citizen', 'place_of_birth', 'tax_residence', 'tax_identification_number'];
-    const force_update_fields       = ['tax_residence', 'tax_identification_number'];
+    const force_update_fields       = ['tax_residence', 'tax_identification_number'].concat(changeable_fields);
+    const has_changeable_fields     = Client.hasCostaricaAccount() && /^CR/.exec(Client.get('loginid')) && !is_fully_authenticated && !is_virtual;
```

Comment:
this isn't global, but you're using it globally?

Reply:
IMO we shouldn't be using `/^CR/.exec(Client.get('loginid'))`, we try not to hardcode login id as much as possible. just checking that current landing company shortcode is costarica can replace `Client.hasCostaricaAccount() && /^CR/.exec(Client.get('loginid'))`

Reply:
then we can remove `!is_virtual` as well

Reply:
Consider it done. Regarding global, I didn't get what you mean. Could you elaborate on that?